src/main/java/com/google/api/gax/grpc/PageStreamingCallSettings.java

```diff
@@ -28,12 +31,12 @@
     ManagedChannel channel = serviceSettingsBuilder.getOrBuildChannel();
```

Comment:
Put this common setup code in one place. (the setup for channel, executor, retryableCodes, retrySettings, and bind)